packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -10,13 +10,15 @@ spec:
       name: {{ .Release.Name }}-rke2-journald-aggregator
   template:
     metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/loggings/rke2/configmap.yaml") . | sha256sum }}
```

Comment:
What does this do? It appears to hash the RKE2 ConfigMap contents and use it as a checksum?

Reply:
It forces a redeploy of the aggregator pods whenever the configuration is updated.  Older versions of logging would not restart these pods and required the users to manually redeploy the daemonsets.

Reply:
Smart... love it.